### PR TITLE
Add initial files.

### PR DESCRIPTION
**Pull request by davidlehn**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+'use strict';
+
+export {default as BrQTable} from './BrQTable.vue';
+export {default as BrQModal} from './BrQModal.vue';
```
Comment (davidlehn) [inline index.js]: I imagine this list will grow so sorting would be good:
```suggestion
export {default as BrQModal} from './BrQModal.vue';
export {default as BrQTable} from './BrQTable.vue';
```

File: index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+'use strict';
```
Comment (davidlehn) [inline index.js]: Don't need this in module code.

**Comment by davidlehn** [comment]
Is this ready to release?

**Review by dlongley — APPROVED** [review]

**Comment by mattcollier** [comment]
@gannan08 @JoshDunnDev @JSAssassin Can you all take a look at this and give a review?

@JSAssassin I know you worked with Josh on designing a component or two, but I would like to ensure that you have a good understanding about what's going on here.

File: .github/workflows/main.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+name: Lint CI
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [12.x]
```
Comment (JSAssassin) [inline .github/workflows/main.yml]: should this be 14.x?

File: BrQModal.vue [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+      if(!$event) {
+        await this.$emitExtendable('close');
+      }
+    },
+    hide() {
+      if(this.$refs.dialog) {
+        this.$refs.dialog.hide();
+      }
+    },
+    show() {
```
Comment (JSAssassin) [inline BrQModal.vue:155]: where is show( ) being used?

File: BrQTitleCard.vue [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    <q-separator />
+    <q-card-section class="q-pa-none">
+      <slot name="body" />
+    </q-card-section>
+  </q-card>
+</template>
+
+<script>
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
```
Comment (JSAssassin) [inline BrQTitleCard.vue:22]: Shouldn't all copyright year be the same (i.e 2020), since it is the initial version?

File: BrQModal.vue (hunk 4 above)
Comment (davidlehn) [reply BrQModal.vue:155]: I'm guessing that's an external API to use if showing/hiding programatically?

File: BrQTitleCard.vue (hunk 5 above)
Comment (davidlehn) [reply BrQTitleCard.vue:22]: Some of the code here was copied from other projects which existed earlier.  For new code we should start with 2020.  @JoshDunnDev perhaps you know which code is new vs old?

File: BrQModal.vue (hunk 4 above)
Comment (aljones15) [reply BrQModal.vue:155]: Maybe we need to include documentation explaining how to use the the `$refs` from a parent component to call `show` etc.?

https://www.codimth.com/blog/web/vuejs/how-use-template-refs-vuejs

it looks like you can do this:

```
<template>
  <br-q-modal ref="myModal" />
</template>

<script>
export default {
  name: 'Bar',
  methods: {
    show() {
      this.$refs.myModal.show();
    }
  }
}
</script>
```

p.s. adding docs can be made an issue and done when ready.

**Comment by davidlehn** [comment]
I know there was some approval for this, but since some time has passed, is this still ready for a release or has anyone come up with a better way to package a kitchen sink of components?

**Pull request merged**